internal/bundle/bundle.go

```diff
@@ -0,0 +1,33 @@
+package bundle
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const MetadataFile = "bundle.yaml"
```

Comment:
This may be not an issue, but if users define their Bundle via a bundle.yaml file somewhere in their source of content it may collide with the metadata file name? Does having something like `metadata.yaml` or `bundle-metadata.yaml` prevent any sorts of collisions in the future?

Reply:
IMO, `bundle.yaml` seems fine. `metadata.yaml` seems more likely to conflict, and `bundle-metadata.yaml` feels a little forced.

If any projects _do_ end up needing a separate unrelated `bundle.yaml` in their project, the advice I'd give would be to put the bundle in a subdirectory. And I think that would be my advice anyway.

How often do you see a project that is packaged via helm intermingle their chart directory in the root of the project? In pretty much every project I've seen, the helm chart is in a subdirectory. Same concept here, IMO.